rule_packages/c/Contracts1.json

```diff
@@ -0,0 +1,48 @@
+{
+  "CERT-C": {
+    "ENV30-C": {
+      "implementation_scope": {
+        "description": "None."
+      },
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Return value of getenv and similar functions cannot be modified.",
+          "kind": "path-problem",
+          "name": "Do not modify the return value of certain functions",
+          "precision": "very-high",
+          "severity": "error",
```

Comment:
I have been using warning for undefined behaviour, although I do see a [mix](https://github.com/github/codeql-coding-standards/blob/main/rule_packages/c/Banned.json#L73) [of](https://github.com/github/codeql-coding-standards/blob/main/rule_packages/c/Banned.json#L111) [examples](https://github.com/github/codeql-coding-standards/blob/main/rule_packages/cpp/Const.json#L17) for UB severity categories in other jsons...